assets/js/leaves.js

```diff
@@ -138,6 +144,15 @@ class Leaf {
     }
   }
 
+  static selectLeavesByTag() {
+    // If the page is loaded with a tag link, select those leaves
+    if (location.search.startsWith("?tag=")) {
```

Comment:
This is order dependent and we might add other params; let's parse as a `URL` first so we can check with `params` object